GameEngine/GameEngine.cpp

```diff
@@ -142,7 +145,7 @@ void GameEngine::reinforcementPhase() {
         int troops = (playerArray[i]->getSizeOfToDefend() / 3) + 3;
         
         //checks if owns entire continents if so add continent bonus to troops
-        vector<Continent*> continents = playerArray[i]->getMap()->getContinents();
+        vector<Continent*> continents = playerArray[i]->getMap().getContinents();
```

Comment:
map is a pointer needs to be -> i think u reveretd the change by accident

Reply:
plz check this and pull from main again to make sure that ur code is the most updated one and u are not reverting any recent changes